packages/bits/spec/components/menu/menu.e2e.ts

```diff
@@ -130,22 +47,7 @@ describe("USERCONTROL Menu", () => {
     });
 
     describe("> key navigation", () => {
-        const checkElementsActiveState = async () => {
-            await menu.toggleMenu();
-            await browser.actions().sendKeys(protractor.Key.ARROW_DOWN).perform();
-            expect(await menu.getMenuItemByIndex(0).isActiveItem()).toBe(true);
-            await browser.actions().sendKeys(protractor.Key.ARROW_DOWN).perform();
-            expect(await menu.getMenuItemByIndex(1).isActiveItem()).toBe(true);
-            await browser.actions().sendKeys(protractor.Key.ARROW_DOWN).perform();
-            expect(await menu.getMenuItemByIndex(2).isActiveItem()).toBe(true);
```

Comment:
I think that only last one should be removed

Reply:
I think Ivan may have meant that the test itself shouldn't be deleted, but we should keep it and just delete lines 139 and 140. Do I have that right, Ivan? But, maybe there is a visual test that covers the whole scenario?

Reply:
Hi Blake, sorry, we discussed it offline with Ivan, and I didn't write it. Yes, the visual test covers the whole scenario anyway.